pytorch_lightning/loops/fit_loop.py

```diff
@@ -205,7 +205,8 @@ def reset(self) -> None:
     def on_run_start(self) -> None:  # type: ignore[override]
         """Calls the ``on_train_start`` hook."""
         # reset train dataloader and val dataloader
-        self.trainer.reset_train_val_dataloaders(self.trainer.lightning_module)
```

Comment:
This method is now unused. Deprecate it?

Reply:
follow up PR! Will be quick if this PR get's merged quickly today.